extensions/parquet-extensions/src/main/java/io/druid/data/input/ParquetHadoopInputRowParser.java

```diff
@@ -0,0 +1,40 @@
+package io.druid.data.input;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.druid.data.input.impl.InputRowParser;
+import io.druid.data.input.impl.ParseSpec;
+
+public class ParquetHadoopInputRowParser implements InputRowParser<InputRow>
```

Comment:
`InputRowParser` as my understanding is more likely a mapper in MapReduce, and the formatted record it processed is guaranteed by the implemetation of `FileInputFormat`. Most of the work of combine each part of data is done in `DruidParquetInputFormat`.